impeller/display_list/aiks_dl_basic_unittests.cc

```diff
@@ -762,5 +775,290 @@ TEST_P(AiksTest, SolidColorCirclesOvalsRRectsMaskBlurCorrectly) {
   ASSERT_TRUE(OpenPlaygroundHere(dl));
 }
 
+TEST_P(AiksTest, CanRenderClippedBackdropFilter) {
+  DisplayListBuilder builder;
+
+  builder.Scale(GetContentScale().x, GetContentScale().y);
+
+  // Draw something interesting in the background.
+  std::vector<DlColor> colors = {DlColor::RGBA(0.9568, 0.2627, 0.2118, 1.0),
+                                 DlColor::RGBA(0.1294, 0.5882, 0.9529, 1.0)};
+  std::vector<Scalar> stops = {
+      0.0,
+      1.0,
+  };
+  DlPaint paint;
+  paint.setColorSource(DlColorSource::MakeLinear({0, 0},              //
+                                                 {100, 100},          //
+                                                 2,                   //
+                                                 colors.data(),       //
+                                                 stops.data(),        //
+                                                 DlTileMode::kRepeat  //
+                                                 ));
```

Comment:
Argument comments would help for the literals here.

Reply:
Done